ConsoleBank/ConsoleBank/Accounting/Account/Transfer/AccountTransferService/AccountTransferService.cs

```diff
@@ -62,50 +55,44 @@ public async Task Transfer(AccountTransferOptions options)
             });
         }
 
-        private async Task RequestTransfer(AccountTransferOptions options,
-            Guid transactionId, decimal withdrawAmount, decimal acquireAmount)
+        private async Task TryTransfer(AccountTransferOptions options,
+            decimal withdrawAmount, decimal acquireAmount, Guid transactionId)
```

Comment:
когда много параметров, складывай в столбик
```suggestion
        private async Task TryTransfer(
            AccountTransferOptions options,
            decimal withdrawAmount, 
            decimal acquireAmount, 
            Guid transactionId)
```